app/flux/App.js

```diff
@@ -0,0 +1,71 @@
+// var API = require('./API');
+
+var React = require('react');
+var Router = require('react-router');
+
+var RouteHandler = Router.RouteHandler;
+
+var Footer = require('./components/Footer.react');
+var Header = require('./components/Header.react');
+
+var UserAPI = require('./API/UserAPI');
+var UserStore = require('./stores/UserStore');
+
+function getStateFromStores() {
+  return {
+    user: UserStore.user
+  }
+}
+
+class App extends React.Component {
```

Comment:
I'm personally really against the class syntax with React right now, there's no good way to specify mixins and the benefit from using them, especially PureMixin which you should be using here far outweighs the minor syntactic sugar it gives.